lib/propshaft/load_path.rb

```diff
@@ -22,7 +22,10 @@ def assets(content_types: nil)
   def manifest
     Hash.new.tap do |manifest|
       assets.each do |asset|
-        manifest[asset.logical_path.to_s] = asset.digested_path.to_s
+        manifest[asset.logical_path.to_s] = {
+          digested_path: asset.digested_path.to_s,
+          integrity: asset.integrity
+        }.stringify_keys
```

Comment:
Why would the keys not already be strings if you're using #to_s on the logical path?

Reply:
Without `stringify_keys` the test below needs to use a symbol for `digested_path`. I thought looked weird having to know that the first key was a string and the second was a symbol

```ruby
test "manifest" do
    @load_path.manifest.tap do |manifest|
      assert_equal "one-f2e1ec14d6856e1958083094170ca6119c529a73.txt", manifest["one.txt"]["digested_path"]
      assert_equal "nested/three-6c2b86a0206381310375abdd9980863c2ea7b2c3.txt", manifest["nested/three.txt"]["digested_path"]
    end
  end
```

Reply:
But isn't this hash static?

```ruby
       manifest[asset.logical_path.to_s] = {
          "digested_path" => asset.digested_path.to_s,
          "digested_path" => asset.integrity
        }
```

would wold in the same way, with no runtime cost.

Reply:
Fixed. My bad, I'm so used to symbols everywhere and stringify for external services that I tend to forget the fat arrow.